src/Compilers/CSharp/Portable/Binder/Binder_Expressions.cs

```diff
@@ -673,6 +673,9 @@ private BoundExpression BindExpressionInternal(ExpressionSyntax node, BindingDia
                 case SyntaxKind.NullLiteralExpression:
                     return BindLiteralConstant((LiteralExpressionSyntax)node, diagnostics);
 
+                case SyntaxKind.UTF8StringLiteralExpression:
```

Comment:
fyi, in the raw-string-work we decided to not have a special expression type.  it's just a string-literal that consumer can check the token kind on. up to you if you want to do the same.

Reply:
> fyi, in the raw-string-work we decided to not have a special expression type.  it's just a string-literal that consumer can check the token kind on. up to you if you want to do the same.

I will add a PROTOTYPE comment for now and will revisit this once raw-string-work gets into this branch.